README.md

```diff
@@ -119,33 +122,34 @@ Below we will give a quick tutorial of how to run atm on your desktop. We will u
    Judgment metric (f1): 0.536 +- 0.067
    Best so far (classifier 21): 0.716 +- 0.035
    ```
-   Occassionally, a worker will encounter an error in the process of building and testing a classifier. When this happens, the worker will print error data to the terminal, log the error in the database, and move on to the next classifier. 
+   Occasionally, a worker will encounter an error in the process of building and testing a classifier. When this happens, the worker will print error data to the terminal, log the error in the database, and move on to the next classifier.
+
+And that's it! You can break out of the worker with <kbd>Ctrl</kbd>+<kbd>c</kbd> and restart it with the same command; it will pick up right where it left off. You can also run the command simultaneously in different terminals to parallelize the work -- all workers will refer to the same ModelHub database. When all 100 classifiers in your budget have been built, all workers will exit gracefully.
 
-AND that's it! You can break out of the worker with Ctrl+C and restart it with the same command; it will pick up right where it left off. You can also run the command simultaneously in different terminals to parallelize the work -- all workers will refer to the same ModelHub database. When all 100 classifiers in your budget have been built, all workers will exit gracefully.
- 
 ## Customizing ATM's configuration and using your own data
 
-ATM's default configuration is fully controlled by the yaml files in ``atm/conig/templates/``. Our documentation will cover the configuration in more detail, but this section provides a brief overview of how to specify the most important values.
+ATM's default configuration is fully controlled by the YAML files in ``atm/config/templates/``. Our documentation will cover the configuration in more detail, but this section provides a brief overview of how to specify the most important values.
```

Comment:
Don't think this is true -- the defaults are now defined by the default values in atm/config.py. We should change this.

Reply:
Sure, I will update this